localgraphclustering/ncpplots.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import pandas as pd
 from plotly.offline import download_plotlyjs, init_notebook_mode, plot, iplot
-import plotly.plotly as py
+import chart_studio.plotly as py
```

Comment:
We really don't want ChartStudio -- this is just for online figures... plotly has all the offiline stuff now.